### DISCUSSION TITLE
School doesn't provide emails to students

### DISCUSSION THREAD

**Discussion by Pushkar-Gooljar** [community/community#65101] [GitHub Education]
### Hi everyone,

I had previously applied for student pack using my student ID as my school doesn't provide emails. The submission form didn't say anything about email back then. But now my student pack expired and when I try repeating what I did this time it is asking for a school email and doesn't even let me use a school ID instead. I issued a ticket about it but it was unanswered after several weeks. I chose other schools in the application form and they don't require emails, only mine does. I really want to use the perks of student pack someone please help me.

**Comment by lab596** [comment]
From my experience there are two methods:

1. Try using a personal email. Using a school-issued email allows for verification faster however a personal email can still be used to verify GitHub Education if the school allows it. 

However, if there is an error message in yellow that states `Please select a different email address` and provides an email tail that is not issued to you, I suggest that you reach out to GitHub Support. 

2. Reach out to GitHub Support directly to explain the situation and provide them with the details of your university's student ID. They should be able to assist you in getting student ID support for the GitHub Student Pack program. You can contact GitHub Support through their [official support page](https://support.github.com/contact).
